presto-main/src/main/java/com/facebook/presto/operator/GroupByIdBlock.java

```diff
@@ -232,4 +234,64 @@ public Block getLoadedBlock()
     {
         return block.getLoadedBlock();
     }
+
+    @Override
+    public byte getByteUnchecked(int position)
+    {
+        return ((UncheckedBlock) block).getByteUnchecked(position);
```

Comment:
@tdcmeehan It might be clearer to add a member variable of type UncheckedBlock and move cast to the constructor. What if block is not UncheckedBlock? How is JVM able to avoid casting on every call?